code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -126,10 +126,11 @@
 		return src.close()
 	return src.open()
 
-// this should probably use dump_contents()
 /obj/structure/closet/ex_act(severity, target)
 	contents_explosion(severity, target)

```

Comment:
Oh yeah, I didn't think about that.
But I did say that it was a **Personal Locker** in #7376, so it was locked yeah.